bleak/backends/bluezdbus/scanner.py

```diff
@@ -45,26 +46,21 @@ class BlueZScannerArgs(TypedDict, total=False):
 
 
 class BleakScannerBlueZDBus(BaseBleakScanner):
-    """The native Linux Bleak BLE Scanner.
+    """API for Bleak Bluetooth LE Scanners, BlueZ implementation.
+
+    A BleakScanner can be used as an asynchronous context manager, in which case it will start and stop scanning.
 
-    For possible values for `filters`, see the parameters to the
+    For possible values for ``bluez['filters']``, see the parameters to the
     ``SetDiscoveryFilter`` method in the `BlueZ docs
     <https://git.kernel.org/pub/scm/bluetooth/bluez.git/tree/doc/adapter-api.txt?h=5.48&id=0d1e3b9c5754022c779da129025d493a198d49cf>`_
 
-    Args:
-        detection_callback:
-            Optional function that will be called each time a device is
-            discovered or advertising data has changed.
-        service_uuids:
-            Optional list of service UUIDs to filter on. Only advertisements
-            containing this advertising data will be received. Specifying this
-            also enables scanning while the screen is off on Android.
-        scanning_mode:
-            Set to ``"passive"`` to avoid the ``"active"`` scanning mode.
-        **bluez:
-            Dictionary of arguments specific to the BlueZ backend.
-        **adapter (str):
-            Bluetooth adapter to use for discovery.
+    :param detection_callback: Optional function that will be called each time a device is discovered or advertising data has changed.
```

Comment:
Let's not change the arg style and keep using the [google doc style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).

If we have type hints in the actual method declarations, we should not need to repeat them in the doc strings.

Reply:
Grmpf, I wasn't aware of that style guide (and apparently there were enough things documented in a way that didn't adhere to it that I didn't notice there was a style guide in effect), so I just picked RTD style because I'm more familiar with that. I'll revert.

Reply:
Reverted back to google style.